massdriver-application/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "name" {
+  description = "The name of the application."
+  type        = string
+}
+
+variable "identity" {
+  description = "The identity this application wil assume. Assume Role Policy for IAM ... TODO"
+  type        = any
+}
+
+variable "root_path" {
```

Comment:
Instead of a path, should we just pass the params/connections as an object? That seems less fragile than a path.

Reply:
I don't even want to pass it at all, but it breaks CI if we don't. Its a lot of boilerplate if we pass the values, IMO